ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -2313,6 +2316,27 @@ export const testAutocomplete = (): Promise<boolean> =>
 export const testPadding = (): Promise<boolean> =>
   new Promise(async (resolve, reject) => {
     try {
+      const divider = createDivider();
```

Comment:
divider is a component, you can use that like this       
` {
          type: WhisperComponentType.Divider,
        },`

Might be me not fully understanding the requirement, but why do we want to use this `createDivider()` function to get divider instead of using divider component directly?

Reply:
Just me making it slightly cleaner. No real difference.